netlify.toml

```diff
@@ -16,3 +16,9 @@
   
 [context.deploy-preview]
   command = "go run mage.go -v DeployPreview"
+
+[context.production.environment]
+HUGO_ENV = "production"
+
+[context.deploy-preview.environment]
+HUGO_ENV = "preview"
```

Comment:
I am guessing this would override the `build.environment` env var in line 11 (https://github.com/cncf/tag-contributor-strategy/pull/584/files#diff-ab8f79b68b7adff7a07db953bf453f3c5aa6ade98d2b1b67d8432b36392489edL11) @cjyabraham

Reply:
Yes. I think that line can be deleted.

Reply:
Done.